man/coap-client.txt.in

```diff
@@ -14,21 +14,27 @@ coap-client - CoAP Client based on libcoap
 
 SYNOPSIS
 --------
-*coap-client* [*-A* type1, _type2_ ,...] [*-t* type] [*-b* [num,]size]
-              [*-B* seconds] [*-e* text] [*-f* file] [*-m* method] [*-N*]
-              [*-o* file] [*-P* addr[:port]] [*-p* port] [*-s* duration]
-              [*-O* num,text] [*-T* token] [*-v* num] [*-a* addr] [*-U*] URI
+*coap-client* [*-a* addr] [*-b* [num,]size] [*-c* certfile] [*-e* text]
+              [*-f* file] [*-k* key] [*-l* loss] [*-m* method] [*-o* file]
+              [*-p* port] [*-r*] [*-s* duration] [*-t* type]  [*-u* user]
+              [*-v* num] [*-A* type1, type2 ,...] [*-B* seconds] [*-N*]
```

Comment:
See my comment on the option `-A` above.

Reply:
Updated